openslides/core/static/js/core/base.js

```diff
@@ -953,6 +948,33 @@ angular.module('OpenSlidesApp.core', [
     }
 ])
 
+/* Converts a snake-case string to CamelCase. */
+.factory('CamelCase', [
+    function () {
+        return function (str) {
+            return str.replace(/-([a-z])/g, function (g) {
```

Comment:
The variable `g` is not self-explaining. Maybe use something like `fragment` or `part` or `match`